compliance_checker/cfutil.py

```diff
@@ -52,6 +54,46 @@
     'atmosphere_sleve_coordinate'
 ]
 
+def check_attr_membership(attr_val, value_set, attr_type=basestring,
```

Comment:
If this is just a helper function, I'd suggest changing the name away from the ```check_``` convention we've reserved for checks in the check classes.

Reply:
Good point, will fix.